demos/compose/voting-app/docker-compose.yml

```diff
@@ -20,6 +20,8 @@ services:
   db:
     container_name: db
     image: postgres:9.4
+    environment:
```

Comment:
can we have a comment as to why this workaround is needed - otherwise it'll become one of those legacy things that no-one is sure whether it's still needed or not.